cloud-consumption/src/main/java/com/sequenceiq/consumption/flow/consumption/storage/handler/StorageConsumptionCollectionHandler.java

```diff
@@ -3,30 +3,48 @@
 import static com.sequenceiq.consumption.flow.consumption.storage.event.StorageConsumptionCollectionHandlerSelectors.STORAGE_CONSUMPTION_COLLECTION_HANDLER;
 import static com.sequenceiq.consumption.flow.consumption.storage.event.StorageConsumptionCollectionStateSelectors.SEND_CONSUMPTION_EVENT_EVENT;
 
+import javax.inject.Inject;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
 
 import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.consumption.converter.CredentialToCloudCredentialConverter;
+import com.sequenceiq.consumption.domain.Consumption;
+import com.sequenceiq.consumption.dto.Credential;
 import com.sequenceiq.consumption.flow.consumption.storage.event.StorageConsumptionCollectionEvent;
+import com.sequenceiq.consumption.flow.consumption.storage.event.StorageConsumptionCollectionHandlerEvent;
+import com.sequenceiq.consumption.service.CredentialService;
 import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
 
 @Component
 public class StorageConsumptionCollectionHandler  extends AbstractStorageOperationHandler {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(StorageConsumptionCollectionHandler.class);
 
+    @Inject
+    private CredentialService credentialService;
+
+    @Inject
+    private CredentialToCloudCredentialConverter credentialConverter;
+
     @Override
-    public Selectable executeOperation(HandlerEvent<StorageConsumptionCollectionEvent> event) throws Exception {
-        StorageConsumptionCollectionEvent data = event.getData();
+    public Selectable executeOperation(HandlerEvent<StorageConsumptionCollectionHandlerEvent> event) throws Exception {
+        StorageConsumptionCollectionHandlerEvent data = event.getData();
+
+        Consumption consumption = data.getContext().getConsumption();
+        String environmentCrn = consumption.getEnvironmentCrn();
+        LOGGER.debug("Getting credential for environment with CRN [{}].", environmentCrn);
+        Credential credential = credentialService.getCredentialByEnvCrn(environmentCrn);
+        credentialConverter.convert(credential);
```

Comment:
No return value is used, I assume because of checkstyle.

Reply:
Yes, it's because of checkstyle, the next PR, that implements the business logic in this handler, will use the returned CloudCredential